sc/src/main/webapp/WEB-INF/jsp/tiles/views/home.jsp

```diff
@@ -1,35 +1,39 @@
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <%@ taglib uri="http://tiles.apache.org/tags-tiles" prefix="tiles" %>
 
-<div class="row margin-bottom-xlarge margin-top-large">
-    <div class="small-12 columns">
-        <tiles:insertAttribute name="home-hero"/>
-    </div>
+<div class="row column margin-bottom-xlarge">
+    <tiles:insertAttribute name="home-hero"/>
 </div>
 
-<div class="row margin-bottom-xlarge">
-    <div class="small-12 columns">
-        <tiles:insertAttribute name="experiment-list-latest-box"/>
-    </div>
+<div class="row column margin-bottom-xlarge">
+    <tiles:insertAttribute name="experiment-list-latest-box"/>
 </div>
 
-<div class="row">
-    <div class="small-12 columns">
-        <tiles:insertAttribute name="search"/>
-    </div>
+<div class="row column margin-bottom-xlarge">
+    <tiles:insertAttribute name="search"/>
 </div>
 
-<%--<div class="row margin-top-xlarge">--%>
-    <%--<tiles:insertAttribute name="type-cell"/>--%>
-    <%--<tiles:insertAttribute name="type-tissue"/>--%>
+<%--<div class="row column">--%>
+    <%--<div class="small-6 columns">--%>
+        <%--<tiles:insertAttribute name="type-cell"/>--%>
+    <%--</div>--%>
+    <%--<div class="small-6 columns">--%>
+        <%--<tiles:insertAttribute name="type-tissue"/>--%>
+    <%--</div>--%>
 <%--</div>--%>
 
-<div class="row margin-top-xlarge">
+<div class="row column margin-bottom-xlarge">
     <tiles:insertAttribute name="publications-box"/>
 </div>
 
-<div class="row expanded jumbo-news-container padding-top-large padding-bottom-large">
-     <div class="small-centered small-8 medium-4 columns">
-         <tiles:insertAttribute name="news"/>
-     </div>
+<div class="row expanded text-center">
+    <div class="small-centered small-8 medium-4 columns padding-top-large padding-bottom-large">
+        <tiles:insertAttribute name="news"/>
+    </div>
 </div>
+
+<script>
+    document.addEventListener("DOMContentLoaded", function(event) {
+        document.getElementById("local-nav-home").className += ' active';
```

Comment:
I alse tried to look under the boot with little success. I’ll raise an issue in their GitHub repo, but it could be that they’re setting manually with their backend on each page (it wouldn’t surprise me).

Reply:
https://github.com/ebiwd/EBI-Framework/issues/136